Editor/testing/luna_tester_engine.cpp

```diff
@@ -616,6 +624,75 @@ void LunaTesterEngine::gameReadyReadStandardOutput()
     }
 }
 
+void LunaTesterEngine::sendPlacingBlock(const LevelBlock &block)
+{
+    if(!isEngineActive())
+        return;
+    LevelData buffer;
+    FileFormats::CreateLevelData(buffer);
+    buffer.blocks.push_back(block);
+    buffer.layers.clear();
+    buffer.events.clear();
+    QString encoded;
+    if(FileFormats::WriteExtendedLvlFileRaw(buffer, encoded))
+        sendItemPlacing(encoded, PendC_SendPlacingItem);
+}
+
+void LunaTesterEngine::sendPlacingNPC(const LevelNPC &npc)
+{
+    if(!isEngineActive())
+        return;
+    LevelData buffer;
+    FileFormats::CreateLevelData(buffer);
+    buffer.npc.push_back(npc);
+    buffer.layers.clear();
+    buffer.events.clear();
+    QString encoded;
+    if(FileFormats::WriteExtendedLvlFileRaw(buffer, encoded))
+        sendItemPlacing(encoded, PendC_SendPlacingItem);
+}
+
+void LunaTesterEngine::sendPlacingBGO(const LevelBGO &bgo)
+{
+    if(!isEngineActive())
+        return;
+    LevelData buffer;
+    FileFormats::CreateLevelData(buffer);
+    buffer.bgo.push_back(bgo);
+    buffer.layers.clear();
+    buffer.events.clear();
+    QString encoded;
+    if(FileFormats::WriteExtendedLvlFileRaw(buffer, encoded))
+        sendItemPlacing(encoded, PendC_SendPlacingItem);
+}
+
+bool LunaTesterEngine::sendItemPlacing(const QString &rawData, PendingCmd ipcPendCmd)
+{
+    //{"jsonrpc": "2.0", "method": "sendItemPlacing", "params":
+    //   {"sendItemPlacing": <RAW ITEM DATA> }}
+    
```

Comment:
Please add the check:
```cpp
if(!m_caps.ipcCommands.contains("sendItemPlacing"))
    return false;
```
In order to don't even try to call this function when using incompatbile LunaLua build.
```